### DISCUSSION TITLE
A pattern to do soft deletions using `Patch` in Refine

### DISCUSSION THREAD

**Discussion by vinclou** [refinedev/refine#2548] [Q&A] [answered]
#### Is there an established pattern to do soft deletions in Refine where you would use PATCH req instead of DELETE?
`Can't really find this in the docs`

##### If not what're recommended ways to implement one, without detouring too much from Refine paradigms?
`Clarification: I'm refactoring an old React app and I can pretty much port the old logic, while just swapping my react-query hook, just wanted to double check if there's anything similar to use in Refine before I commit to it 100%`

**Comment by salihozdemir** [comment, marked as answer]
Hey @vinclou,
If you are using an out-of-box refine data provider, you can see how to override the data provider in the link below.
https://refine.dev/docs/faq/#how-i-can-override-specific-function-of-data-providers

If you are create a new own data provider, you can determine to use which req with `metaData`.

So of corse you can do it.
